networking.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: page
+title: Making Networking Requests in Flutter
+permalink: /networking/
+---
+
+* TOC Placeholder
+{:toc}
+
+## Using the `http` package
+
+Flutter supports the the [http package][http], version `0.11.3+12` or later.
+
+Start by declaring a dependency by adding an http line under `dependencies` in
+`pubspec.yaml`:
+
+```
+dependencies:
+  ...
+  http: '>=0.11.3+12'
+```
+
+## Making requests
+
+You can create an HTTP [Client][client] with the `Client` constructor:
+
+<!-- skip -->
+```dart
+import 'package:http/http.dart' as http;
+
+http.Client _httpClient = new http.Client();
+```
+
+The client supports common HTTP operations:
+
+* **HTTP GET:** Use [`get`][get] for a general request, [`read`][read] for a
+ request returning a string, or [`readbytes`][readbytes] for a request
+ returning bytes.
+
+* **HTTP POST:** Use [`post`][post] for a general post.
+
+Note that the API uses [Dart
+Futures](https://www.dartlang.org/tutorials/language/futures) in the return
+values. These are most easily used with the `then()` methods like this:
```

Comment:
I think we want to prefer showing async/await syntax. More modern, and it makes for easier to read code.